packages/components/stories/Loader.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
+import { IconsProvider } from '../src';
 
 import { Loader, CIRCULAR_PROGRESS_SIZE as SIZE } from '../src/';
 
+const apps = ['portal', 'tdc', 'tdp', 'tds', 'datastreams', 'tic', 'tmc'];
```

Comment:
Should be retrieved from a constant for once (out of this file)

Reply:
fixed